2-Browsers/Week1/homework/ex2-aboutMe/index.js

```diff
@@ -9,4 +9,15 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 3. Look in the css file!
 ------------------------------------------------------------------------------*/
 
-// TODO add your JavaScript code here.
+
+const nickname = document.getElementById('nickname');
+nickname.textContent = 'Valentyn Genkin';
+
+const favFood = document.querySelector('li').nextElementSibling.children;
+Array.from(favFood).forEach(element => element.textContent = 'Baked chicken');
+
+const homeTown = document.querySelector('ul').children[2].children;
+Array.from(homeTown).forEach(element => element.textContent = 'Rotterdam');
```

Comment:
`document.querySelector` returns a so-called "NodeList", which can you iterate over (go over). So you can directly do `.forEach` on `homeTown` 🙂 See [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll?retiredLocale=nl#accessing_the_matches) for more info.

```suggestion
homeTown.forEach(element => element.textContent = 'Rotterdam');
```

Nice syntax btw! 💪